src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml

```diff
@@ -81,7 +94,12 @@ tests:
           [1657021528.886209][7089:7094] CHIP:TOO:     [23]: 65531
           [1657021528.886234][7089:7094] CHIP:TOO:     [24]: 65532
           [1657021528.886258][7089:7094] CHIP:TOO:     [25]: 65533
-      disabled: true
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [25]
```

Comment:
This does not make sense.  I'd think this should list all the mandatory attribute ids... Listing just "25" is weird because "25" is in fact not an attribute id on this cluster.

Reply:
Fixed